resources/views/email-verification/done.blade.php

```diff
@@ -0,0 +1,22 @@
+@extends('nodes.api::email-verification.base')
+
+@section('content')
+    <div id="nBox" class="panel panel-default nodes-center">
+        <div class="panel-heading">
+            <h3 class="panel-title">Email verification</h3>
+        </div>
+        <div class="panel-body">
+            @if (Session::has('success'))
+                <div class="alert alert-success text-center" role="alert">
+                    {{ Session::get('success') }}
+                </div>
+            @endif
+            <p><strong>Congratulations!</strong></p>
+            <p>Your email has been now been verified.</p>
```

Comment:
Question: Isn't it your entire user you're verifying? Or is it just the e-mail? Just thinking if it should say `Your user has now been verified` instead? :)